backend/src/index.ts

```diff
@@ -1,12 +1,22 @@
 import express, { Request, Response, Application } from "express"
 import dotenv from "dotenv"
 import exampleRouter from "./routes/example.routes"
+import { Client } from "pg"
 
 // For env File
 dotenv.config()
 
 const app: Application = express()
 const port = process.env.PORT ?? 8000
+const db = new Client({
+  user: process.env.PG_USER,
+  host: process.env.PG_HOST,
+  database: process.env.PG_DATABASE,
+  port: parseInt(process.env.PG_PORT ?? "5432"),
+  password: process.env.PG_PASSWORD
+})
+
+db.connect()
```

Comment:
Please don't connect to db in index.ts kub,better move to repository

Reply:
JK, I think move to config/pgdbConnnection.ts is better

Reply:
Why shouldn't connect in index.ts? and if moved to repo isn't it will duplicate code of connecting the db?

Reply:
//You will need to call it in index.ts anyways but please move the function to config

Reply:
Not sure but I believe that we will need to use db variable in the repository too, maybe for querying? Anyways it makes the code a little bit less coupling